manifests/params.pp

```diff
@@ -99,6 +99,8 @@
   $max_keep_alive = 10000
   $num_workers = min($::processorcount, 8)
 
+  $puppet_wsgi_processes = 3
```

Comment:
its the default value  pulp provides:  https://github.com/pulp/pulp_puppet/blob/master/pulp_puppet_plugins/etc/httpd/conf.d/pulp_puppet.conf#L39